FDB failure 
```
[2026-03-25T12:14:39.982Z] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M3:enforce (ban-duplicated-classes) on project kie-wb-common-ala-wildfly-provider: Some Enforcer rules have failed. Look above for specific messages explaining why the rule failed. -> [Help 1]
[2026-03-25T12:14:39.982Z] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M3:enforce (ban-duplicated-classes) on project kie-wb-common-ala-wildfly-provider: Some Enforcer rules have failed. Look above for specific messages explaining why the rule failed.
[2026-03-25T12:14:39.982Z]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
[2026-03-25T12:14:39.982Z]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
```

--- a/kie-wb-common-ala/kie-wb-common-ala-services-rest/pom.xml
+++ b/kie-wb-common-ala/kie-wb-common-ala-services-rest/pom.xml
@@ -90,6 +90,15 @@
           <groupId>javax.ws.rs</groupId>
           <artifactId>javax.ws.rs-api</artifactId>
         </exclusion>
+        <!-- Exclude old Bouncy Castle versions from docker-client shaded jar -->
+        <exclusion>
+          <groupId>org.bouncycastle</groupId>
+          <artifactId>bcprov-jdk15on</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.bouncycastle</groupId>
+          <artifactId>bcpkix-jdk15on</artifactId>
+        </exclusion>
       </exclusions>
       <scope>test</scope>
     </dependency>

--- a/kie-wb-common-ala/kie-wb-common-ala-wildfly/kie-wb-common-ala-wildfly-provider/pom.xml
+++ b/kie-wb-common-ala/kie-wb-common-ala-wildfly/kie-wb-common-ala-wildfly-provider/pom.xml
@@ -135,6 +135,15 @@
           <groupId>org.sonatype.plexus</groupId>
           <artifactId>plexus-sec-dispatcher</artifactId>
         </exclusion>
+        <!-- Exclude old Bouncy Castle versions to prevent duplicate classes -->
+        <exclusion>
+          <groupId>org.bouncycastle</groupId>
+          <artifactId>bcprov-jdk15on</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.bouncycastle</groupId>
+          <artifactId>bcpkix-jdk15on</artifactId>
+        </exclusion>
       </exclusions>
     </dependency>
     <dependency>

